src/Logging/DebugViewFriendlyErrorFormatter.php

```diff
@@ -127,9 +128,22 @@ public function output($statusCode)
         $output = $renderer->renderHeader();
         $output .= $renderer->renderInfo("Website Error", $this->getTitle(), $this->getBody());
 
-        if (Email::config()->admin_email) {
-            $mailto = Email::obfuscate(Email::config()->admin_email);
-            $output .= $renderer->renderParagraph('Contact an administrator: ' . $mailto . '');
+        if ($adminEmail = Email::config()->admin_email) {
+            if (is_array($adminEmail)) {
+                if (count($adminEmail) > 0) {
+                    $email = array_keys($adminEmail)[0];
+                    $name = array_values($adminEmail)[0];
+
+                    $output .= $renderer->renderParagraph(sprintf(
+                        'Contact %s: %s',
+                        Convert::raw2xml($name),
```

Comment:
There's a bit too much nesting here for my liking - any chance of a private method with early returns, like `$this->getAdminEmails()` and `$this->appendContactInformation($emails)`?

Reply:
done